pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -23,6 +23,7 @@ type ExplainerConfig struct {
 
 	DefaultImageVersion  string   `json:"defaultImageVersion"`
 	AllowedImageVersions []string `json:"allowedImageVersions"`
+	NumWorkers           int      `json:"numWorkers"`
```

Comment:
As discussed in the WG, can we call this parallelization in the TensorflowSpec, XGBoostSpec, or whereever else relevant?

Reply:
Changed to `parallellism`.